tests/transactional/transactional_update.pm

```diff
@@ -76,6 +77,8 @@ sub run {
 
     # Find snapshot number for rollback
     my $snap = script_output "snapper list | tail -1 | cut -d'|' -f1 | tr -d ' *'";
+    # For s390x we use only 1 console that can be polluted by btrfs kernel messages
```

Comment:
Shouldn't those messages be turned off?

Reply:
They always appear when `snapper list` is executed.
https://progress.opensuse.org/issues/93125#note-9

Reply:
`qgroup rescan init failed, qgroup is not enabled` seems to be a error which needs some investigation. qgroups should be enabled...

Reply:
Sorry, I have mixed it a bit. DVD installation seems to have `qgroups` enabled, however the `sclp` console is polluted by [[   50.806889] BTRFS info (device vda2): qgroup scan completed (inconsistency flag cleared)](https://openqa.suse.de/tests/6325012#step/transactional_update/52) message, therefore `$snap` variable stores not only the snapshot number, but also the message and fails over [here](https://openqa.suse.de/tests/6325012#step/transactional_update/249). In case of s390x Image, it shows the above [failure message](https://openqa.suse.de/tests/6362304#step/transactional_update/52).

The s390x image has a different setup compared to QEMU image and s390x DVD installation. I will file a report. 
Thanks for raising the awareness!

Reply:
Hm, that message should ideally not be shown either. Maybe something should set the loglevel explicitly?

Reply:
Is this expected setup for logging ? 

```
s390kvm086:~ # cat /proc/sys/kernel/printk
7       4       1       7
```

For QEMU image the current level is set for warning, if I am not mistaken

```
4	4	1	7
```

Reply:
It seems like the s390x images are missing `quiet` in the kernel cmdline. For some reason one of them specifies `TERM=dumb`, which seems wrong.

Reply:
I think that  `hvc_iucv` is also noop in case z/KVM.
```
hvc_iucv=       [S390]  Number of z/VM IUCV hypervisor console (HVC)
                                terminal devices. Valid values: 0..8
```

https://www.kernel.org/doc/html/latest/admin-guide/kernel-parameters.html

Reply:
Missing `quiet` is the culprit of those messages. Tested on qemu image. Nice one, thanks!

```
localhost login: [   26.406188] BTRFS info (device vda3): qgroup scan completed (inconsistency flag cleared)
[   41.617909] BTRFS info (device vda3): qgroup scan completed (inconsistency flag cleared)
 # | Type   | Pre # | Date                     | User | Used Space | Cleanup | Description           | Userdata
---+--------+-------+--------------------------+------+------------+---------+-----------------------+---------
0  | single |       |                          | root |            |         | current               |         
1  | single |       | Mon Jun 21 06:45:17 2021 | root | 848.00 KiB |         | first root filesystem |         
2  | single |       | Fri Jul  2 07:58:13 2021 | root | 272.00 KiB |         | Snapshot Update of #1 |         
3  | single |       | Fri Jul  2 07:59:10 2021 | root | 288.00 KiB |         | Snapshot Update of #2 |         
4* | single |       | Fri Jul  2 09:27:02 2021 | root | 420.00 KiB |         | Snapshot Update of #3 |  
```

Current printk setup
```
 7	4	1	7
 ```

Reply:
> It seems like the s390x images are missing `quiet` in the kernel cmdline. For some reason one of them specifies `TERM=dumb`, which seems wrong.

I have noticed the `TERM=dumb` only related to z/VM, as well as `hvc_iucv`. https://documentation.suse.com/sles/15-SP3/single-html/SLES-deployment/#sec-appdendix-parm-examples
I guess, SLEM image should run on top z/KVM rather than z/VM.